src/format_string.h

```diff
@@ -45,23 +45,23 @@ class FormatString {
   std::string format_str(std::vector<std::unique_ptr<IPrintable>> &args);
 
   // length returns the length of the format string
-  inline size_t length() const noexcept
```

Comment:
Are these compiler directives considered bad?

Reply:
`inline` doesn't do anything if used on a member defined in the class definition.

https://isocpp.org/wiki/faq/inline-functions#inline-member-fns-more

Reply:
Ohh TIL.